src/bundle/Resources/views/content/content_edit/content_edit.html.twig

```diff
@@ -27,7 +27,10 @@
         </h1>
 
         <div class="small">
-            {{ contentType.name }} / {{ 'created_by'|trans({'%name%': 'Administrator'})|desc('Created by %name%') }} / {{ content.versionInfo.creationDate|localizeddate('medium', 'medium', app.request.locale) }} / {{ 'content_id'|trans({'%contentId%': content.id})|desc('Content ID: %contentId%') }}, {{ 'location_id'|trans({'%locationId%': locationId})|desc('Location ID: %locationId%') }}
+            {{ contentType.name }} /
```

Comment:
I know I saw this multiple times across latest PR's, are you sure this is the only occurrence that need to be changed?

Reply:
There might be something in repository-forms? ping @webhdx

Reply:
I missed edit_edit view, but I added it already.

Reply:
It only applies to `content_edit` and `user_edit` which is already done by @mikadamczyk